parachain/pallets/outbound-queue/runtime-api/src/lib.rs

```diff
@@ -2,11 +2,14 @@
 // SPDX-FileCopyrightText: 2023 Snowfork <hello@snowfork.com>
 #![cfg_attr(not(feature = "std"), no_std)]
 
+use snowbridge_core::outbound::{Fees, Message};
 use snowbridge_outbound_queue_merkle_tree::MerkleProof;
 
 sp_api::decl_runtime_apis! {
 	pub trait OutboundQueueApi
 	{
 		fn prove_message(leaf_index: u64) -> Option<MerkleProof>;
+
+		fn calculate_fee(message: Message) -> Option<Fees<u128>>;
```

Comment:
Should make the fee type a generic parameter, as the balance type is configured in the runtime crate.

```suggestion
		fn calculate_fee(message: Message) -> Option<Fees<Balance>>;
```